src/mmword.c

```diff
@@ -341,70 +338,70 @@ l7220:  i=i+m-1;
 }
 
 void gosub_7320(void) {
-        /* Subroutine:  get next L1_ from original file */
+  // Subroutine:  get next L1_ from original file
```

Comment:
This comment applies to the previous line, and should go before it. Dunno if you are adventurous enough to try automatically detecting it, but a fairly good heuristic would be that the start of the comment is more indented than the following line.

Reply:
Right now I'm leaning more for changing those manually, but I'll give a look and see if they are common enough to be worth automating.